internal/router/middlewares/ratelim_test.go

```diff
@@ -197,36 +74,6 @@ func TestLimit1IP(t *testing.T) {
 	}
 }
 
-func TestRateLim10Addresses(t *testing.T) {
-	t.Parallel()
-
-	// Only allow 150 req per second *per address*.
-	cfg := RateLimiterConfig{
-		Default: RateLimiterRouteConfig{
-			MaxRPI:   150,
-			Interval: time.Second,
-		},
-		JSONRPCRoute: "/rpc",
-	}
-	rlcm, err := RateLimitController(cfg)
-	require.NoError(t, err)
-	rlc := rlcm(dummyHandler{})
-
-	// Do 1000 requests as fast as we can with *different addresses*, and see that
```

Comment:
I'm assuming we are referring to wallet addresses here, since we are looking at `ContextKeyAddress`

Reply:
You're right